config/initializers/core.rb

```diff
@@ -30,6 +30,14 @@
 Core::Registry::Repository[:cms_api] =
   Core::Repository::CMS::CmsApi.new
 
+if Feature.active?(:cms_footer)
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::CMS.new
+else
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::Static.new
+end
+
```

Comment:
Is the footer cached? Otherwise it will be one more CMS request for each page request.